src/tests/ftest/daos_test/suite.yaml

```diff
@@ -22,7 +22,7 @@ timeouts:
   test_daos_capability: 104
   test_daos_epoch_recovery: 104
   test_daos_md_replication: 104
-  test_daos_rebuild_simple: 1800
+  test_daos_rebuild_simple: 2000
```

Comment:
Just pointing out that every time we increase timeouts for these long-running `pr` tests, we slow down CI overall and slow down individual PRs. Last I checked, we have ~4-8 hours of rebuild `pr` tests. It's not feasible to keep increasing the test time for PRs that don't even affect rebuild.

We really need to split some of these `pr` rebuild tests into `pr` and `daily_regression`